src/atldld/utils.py

```diff
@@ -24,10 +24,12 @@
 """
 
 import os
+import pathlib
+from typing import Optional, Union
 
-import matplotlib.pyplot as plt
 import numpy as np
 import requests
+from PIL import Image
 
 CACHE_FOLDER = os.path.expanduser("~/.atldld/")
```

Comment:
We might as well migrate to `pathlib` fully and rewrite this too IMO

Reply:
I think this was done in #57, which was just merged to main and I also updated this branch with these changes in b02a05a. There's no `os.path` in this module any more :)

Reply:
Perfect (resolved)!